### ISSUE TITLE
'quick' tap causes links in following sections to activate.

### ISSUE THREAD

**Issue by kenschultz** [tapquo/Lungo.js#144]
I have three large links contained within an article on the index page, they each link to a section contained in a separate html file. If one of these links are quickly tapped and the following section has a link located in the same touch area that was quickly tapped, it will activate the link in the new section.

This does not occur with a long touch. This looks like the 'quick' tap duration is being extended or has a duration parameter that is to great.

To recap: quick taps are carrying over into following sections and activating links in the same space as the quick tap in the preceding section.

**Comment by kenschultz** [comment]
Maybe something with quo.js - I'm digging around looking for the supporting lines to the quick tap command...

**Comment by soyjavi** [comment]
Try event singleTap() :)
